bref/symfony-bridge/0.1/post-install.txt

```diff
@@ -0,0 +1,8 @@
+<bg=blue;fg=white>                     </>
+<bg=blue;fg=white> Bref Symfony Bridge </>
+<bg=blue;fg=white>                     </>
+
+  * <fg=blue>Read</> the full documentation to properly configure your
+    Symfony application at <comment>https://bref.sh/docs/frameworks/symfony.html</>
```

Comment:
The bridge is not documented there.

We should document it first in the offcial docs as the bridge is starting to get more attention lately and is working well in several projects of mine. 

The other solution is to point to the bridge's README file

Reply:
The official documentation will be there though, once the Symfony Bridge becomes stable. So that link makes sense (the README should be empty eventually).